ag/ag.proto

```diff
@@ -199,6 +199,23 @@ message Issue {
     uint64 issueNumber  = 4;  // Issue number on scm. Needed for associating db issue with scm issue
 }
 
+message PullRequest {
+    enum Stage {
+        NONE    = 0;
+        DRAFT   = 1;
+        REVIEW  = 2;
+        APPROVED = 3;
+    }
+    uint64 ID                   = 1;
+    uint64 externalRepositoryID = 3; // Represents the external repository ID
```

Comment:
`repositoryID`

Reply:
Is that not confusing? repositoryID can mean both the internal repository ID and the external GitHub repository ID.

Reply:
Not sure. Do we actually use the database internal repository ID as a foreign key? I agree that it might be good to distinguish these... I checked:

```proto
message Repository {
...
    uint64 repositoryID   = 3;  // ID of a github repository
```
```proto
message Issue {
    uint64 ID           = 1;
    uint64 repositoryID = 2;  // Represents the internal ID of a repository
```

Maybe we should adopt a convention for this stuff; e.g., to prefix database specific (foreign key) IDs with `dbRespositoryID`... But perhaps we should do that separately in a different PR that fixes all such issues. Alternatively, we could prefix other external IDs with e.g., `scmRepositoryID`. Not sure what makes the most sense. I'll create a separate issue for this.

PS: Noticed also that field number `2` seems to be missing.

Reply:
I will leave things as is for now then, and add a TODO(meling) for reference.
I also fixed the field number in the most recent commit.